components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -6257,6 +6331,47 @@ public int getAPIID(APIIdentifier apiId, Connection connection) throws APIManage
         return id;
     }
 
+    public int getAPIProductID(APIProductIdentifier apiProductId, Connection connection) throws APIManagementException {
```

Comment:
Is it necessary to get a connection as a parameter? We can use `connection = APIMgtDBUtil.getConnection();` Also I am not understand usage of `created` variable.